images/ceph/Makefile

```diff
@@ -89,9 +89,10 @@ generate-csv-templates: $(OPERATOR_SDK) $(YQ) ## Generate CSV templates for OLM
 	@# first, copy the existing CRDs and OLM catalog directory to CSV_TEMPLATE_DIR
 	@# then, generate or copy all prerequisites into CSV_TEMPLATE_DIR (e.g., CRDs)
 	@# finally, generate the templates in-place using CSV_TEMPLATE_DIR as a staging dir
-	@mkdir -p $(CSV_TEMPLATE_DIR)
-	@cp -a ../../cluster $(CSV_TEMPLATE_DIR)/cluster
-	@set -eE;\
+	@ echo yq=$(YQ) operator-sdk=$(OPERATOR_SDK)
+	mkdir -p $(CSV_TEMPLATE_DIR)
+	cp -a ../../cluster $(CSV_TEMPLATE_DIR)/cluster
+	set -eE;\
```

Comment:
Removing `@` symbols here and elsewhere was useful in debugging build issues in the CI for this PR and will be useful in the future as well, so I keep them.

Reply:
How is it useful in debuging? Is it printing the commands?